### DISCUSSION TITLE
How can I get AdministrationSiteType property using PnP?

### DISCUSSION THREAD

**Discussion by PedroMordeP** [pnp/powershell#49] [Q&A] [answered]
After set a site as TenantAdministration is there a way to know if a specific SP2019 site is "TenantAdministration" using PnP?

```
$site = get-spsite -Identity $siteColUrl
$site.AdministrationSiteType = [Microsoft.SharePoint.SPAdministrationSiteType]::TenantAdministration
```

Script found on this [article](https://docs.microsoft.com/en-us/archive/blogs/vesku/provisioning-site-collections-using-sp-app-model-in-on-premises-with-just-csom) to use Get-PnPTenantSite from outside the server.

Is this only available on Get-SPSite properties?
I searched on Get-PnPSite and Get-PnPWeb results properties but can't find it, is there other property that can i check to know this?

**Comment by erwinvanhunen** [comment, marked as answer]
`Get-SPSite` cmdlet only works when effectively running PowerShell on a SharePoint server and use the SharePoint Server Object Model to access server settings. The PnP Cmdlets can only provide what Microsoft gives us as we have to use the Client-Side Object Model which is a subset of the Server Object Model. The CSOM api does not provide this level of access unfortunately. So for SharePoint 2019 you're limited to using the SharePoint Management Shell cmdlets on the server instead. Also using `Get-PnPTenantSite` for on-premises will not allow you to designate the AdministrationSiteType unfortunately. It's simply technically not possible given the APIs we are provided with.
